src/assets/wise5/authoringTool/display-branch-path-steps/display-branch-path-steps.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { DisplayBranchPathStepsComponent } from './display-branch-path-steps.component';
+import { TeacherProjectService } from '../../services/teacherProjectService';
+import { StudentTeacherCommonServicesModule } from '../../../../app/student-teacher-common-services.module';
+import { provideHttpClient } from '@angular/common/http';
+import { provideHttpClientTesting } from '@angular/common/http/testing';
+
+describe('DisplayBranchPathStepsComponent', () => {
+  let component: DisplayBranchPathStepsComponent;
+  let fixture: ComponentFixture<DisplayBranchPathStepsComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [DisplayBranchPathStepsComponent, StudentTeacherCommonServicesModule],
+      providers: [provideHttpClient(), provideHttpClientTesting(), TeacherProjectService]
```

Comment:
provideHttpClientTesting() is not necessary, remove?
```suggestion
      providers: [provideHttpClient(), TeacherProjectService]
```